app/src/main/kotlin/org/gnucash/android/model/Money.kt

```diff
@@ -520,7 +520,7 @@ class Money : Number, Comparable<Money> {
          */
         @JvmStatic
         fun getBigDecimal(numerator: Long, denominator: Long): BigDecimal {
-            var denominator = denominator
+            @Suppress("NAME_SHADOWING") var denominator = denominator
```

Comment:
I changed it to "denom" in another branch

Reply:
I've reverted this change to avoid a conflict.